asab/config.py

```diff
@@ -161,6 +161,10 @@ def __init__(self, config_section_name, config=None):
 
 			# Merge config defaults of each base class in the 'inheritance' way
 			for key, value in base_class.ConfigDefaults.items():
+
+				if value is None:
+					raise ValueError("%s value for key %s in %s" % (value, key, base_class))
```

Comment:
The message in the exception is not describing the nature of the problem.

Reply:
@ateska Could you recommend me a better description then?

Reply:
"Don't use None in ConfigDefaults, found in {}:{}".format(section, key)

Reply:
Example of formatted message

```
None value not allowed in ConfigDefaults. Found in connection:MySQLConnection1:connect_timeout
```